Hearthstone Deck Tracker/Config.cs

```diff
@@ -151,7 +151,7 @@ public class Config
 		[DefaultValue(true)]
 		public bool ConstructedAutoImportNew = true;
 
-		[DefaultValue(true)]
+		[DefaultValue(true)]

```

Comment:
There should not be a change here.

Probably inconsistent line endings :/? I need to reformat everything at some point.